arc/cmd/verify_test.go

```diff
@@ -135,6 +135,98 @@ func Test_VerifyCmd_unknown_verification_alg(t *testing.T) {
 	assert.EqualError(t, err, expectedErr)
 }
 
+func Test_VerifyCmd_missing_header_key(t *testing.T) {
+	cmd := NewVerifyCmd()
+
+	files := []fileEntry{
+		{"ear.jwt", testJWT},
+	}
+	makeFS(t, files)
+
+	args := []string{
+		"ear.jwt",
+	}
+	cmd.SetArgs(args)
+
+	expectedErr := `failed to get JWK key from JWT header`
+
+	err := cmd.Execute()
+	assert.EqualError(t, err, expectedErr)
+}
+
+func Test_VerifyCmd_incorrect_jws(t *testing.T) {
+	cmd := NewVerifyCmd()
+
+	files := []fileEntry{
+		{"ear.jwt", testJWT_JWK[1:]},
+	}
+	makeFS(t, files)
+
+	args := []string{
+		"ear.jwt",
+	}
+	cmd.SetArgs(args)
+
+	expectedErr := `failed to parse serialized JWT: jws.Parse: failed to parse compact format: failed to decode protected headers: failed to decode source: illegal base64 data at input byte 212`
+
+	err := cmd.Execute()
+	assert.EqualError(t, err, expectedErr)
+}
+
+func Test_VerifyCmd_header_key_and_expired_token(t *testing.T) {
+	cmd := NewVerifyCmd()
+
+	files := []fileEntry{
+		{"ear.jwt", testRealmJWT},
+	}
+	makeFS(t, files)
+
+	args := []string{
+		"ear.jwt",
+	}
+	cmd.SetArgs(args)
+
+	expectedErr := `verifying signed EAR from "ear.jwt" using "JWK header" key: failed verifying JWT message: jwt.Parse: failed to parse token: jwt.Validate: validation failed: "exp" not satisfied: token is expired`
```

Comment:
Just a suggestion: the high order bit here is _"validation failed: "exp" not satisfied: token is expired”_

You can use `assert.ErrorContains` with just that substring.